README.md

```diff
@@ -126,7 +126,7 @@ The following table shows which versions of league/commonmark are compatible wit
     </thead>
     <tbody>
         <tr>
-            <td><strong>0.11.1</strong><br>0.11.0</td>
+            <td><strong>0.11.x</strong></td>
```

Comment:
So far all three `0.11` versions are compatible (and the next one should be too) so I'm taking the lazy approach here :P